components/io.asgardio.java.saml.sdk/src/main/java/org/wso2/carbon/identity/sso/agent/sdk/SAML2SSOManager.java

```diff
@@ -88,18 +88,18 @@
 import org.w3c.dom.ls.DOMImplementationLS;
 import org.w3c.dom.ls.LSOutput;
 import org.w3c.dom.ls.LSSerializer;
-import org.wso2.carbon.identity.sso.agent.saml.artifact.SAMLSSOArtifactResolutionService;
-import org.wso2.carbon.identity.sso.agent.saml.bean.LoggedInSessionBean;
-import org.wso2.carbon.identity.sso.agent.saml.bean.SSOAgentConfig;
-import org.wso2.carbon.identity.sso.agent.saml.exception.ArtifactResolutionException;
-import org.wso2.carbon.identity.sso.agent.saml.exception.InvalidSessionException;
-import org.wso2.carbon.identity.sso.agent.saml.exception.SSOAgentException;
-import org.wso2.carbon.identity.sso.agent.saml.internal.SSOAgentServiceComponent;
-import org.wso2.carbon.identity.sso.agent.saml.security.X509CredentialImpl;
-import org.wso2.carbon.identity.sso.agent.saml.session.management.SSOAgentSessionManager;
-import org.wso2.carbon.identity.sso.agent.saml.util.SSOAgentConstants;
-import org.wso2.carbon.identity.sso.agent.saml.util.SSOAgentDataHolder;
-import org.wso2.carbon.identity.sso.agent.saml.util.SSOAgentUtils;
+import org.wso2.carbon.identity.sso.agent.sdk.artifact.SAMLSSOArtifactResolutionService;
```

Comment:
Move to io.asgardio.java.saml.sdk

Reply:
Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/2/commits/ccb42095df08cb3685ad47ae534f9dd826d4670e